util/helpers.ts

```diff
@@ -73,3 +98,58 @@ export const getTwitterHandle = (url?: string) => {
   const twitterHandle = url?.split('/').pop();
   return twitterHandle;
 };
+
+/**
+ * function that returns day, date and key of a given session
+ * @param schedule ISchedule
+ * @returns Array<dates>
+ * @example
+ */
+export const getDaysFromSchedule = (schedule: ISchedule): Array<IDateForDayButton> => {
+  const keys = Object.keys(schedule.data);
+  const datesToSave = keys.map((key, index) => {
+    const date = new Date(key).getDate();
+    const dateWithSuffix = getSuffixForDate(date);
+    return {
+      day: `Day ${index + 1}`,
+      date: `${date}${dateWithSuffix}`,
+      key,
+    };
+  });
+  return datesToSave;
+};
+
+/**
+ * function that returns suffix for a date
+ * @param date number
+ * @returns Array<dates>
+ * @example 2 returns 2nd
+ */
+const getSuffixForDate = (date: number) => {
+  const suffix = date > 0 ? ['th', 'st', 'nd', 'rd'][(date > 3 && date < 21) || date % 10 > 3 ? 0 : date % 10] : '';
+  return suffix;
+};
+
+/**
+ * a function that gets the start time, end time, and room.title of a session from the schedule.data array
+ * @param start_date_time start time of a session
+ * @param end_date_time end time of a session
+ * @param room array of rooms
+ * @returns start time, end time, and room.title
+ * @example getScheduleTimesAndLocation('session-1')  // returns 9:00 AM - 10:00 AM  |  Room 1
+ */
+export const getScheduleTimeAndLocation = (start_date_time: string, end_date_time: string, room?: Room) => {
+  // convert time to 12 hour format and hh:mm aa
+  const startTime = new Date(start_date_time).toLocaleTimeString('en-US', {
+    hour: 'numeric',
+    minute: 'numeric',
+    hour12: true,
+  });
+  const endTime = new Date(end_date_time).toLocaleTimeString('en-US', {
+    hour: 'numeric',
+    minute: 'numeric',
+    hour12: true,
+  });
+
+  return `${startTime} - ${endTime}  |  Room ${room !== undefined ? room.title : ''}`;
```

Comment:
If we cannot find the room, this will currently say `3:40 PM - 4:20 PM | Room ` which looks broken. I'd suggest we rather display `3:40 PM - 4:20 PM | Room TBA` or even not mention the room and simply display `3:40 PM - 4:20 PM`.

Reply:
Ok lemme work on that